bash/cmd-line-searches/howmuch

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
This should be

```
#!/usr/bin/env bash
```

https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash/21613044

Reply:
Sounds good! I was unaware of that convention. I'll change it here, and in all the other bash scripts I have contributed.